### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by tvlooy**
Starting to cleanup and split giant controllers. Some bugs were found and fixed. Needs testing by me.

@acrobat @rvanginneken Good idea to organize the controllers like this? Disadvantage is that you have to specify the template for each Template annotation. What do you think?

Ps: counter-PR's are welcome. I'm just fooling around

### CODE REVIEW

**Comment by acrobat** [comment]
The explicit notation of each template isn't a disadvantage for me as this make it easier to understand for newcomers to symfony/project to see where they can find the templates.

**Comment by tvlooy** [comment]
This is some cleanup and is ready for now. This will make next week a bit easier, now that we have multiple controllers we will be working in the same files less often.

Tested and nothing seems to break. Looking for OK and merge.

File: src/Intracto/SecretSantaBundle/Controller/StartpageController.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+namespace Intracto\SecretSantaBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Intracto\SecretSantaBundle\Form\PoolType;
+use Intracto\SecretSantaBundle\Entity\Pool;
+
+class StartpageController extends Controller
```
Comment (acrobat) [inline src/Intracto/SecretSantaBundle/Controller/StartpageController.php]: Maybe call this `HomepageController`? `StartpageController` sounds a bit weird to me

**Comment by tvlooy** [comment]
yeah, I gave it some thought but naming things is hard ... :-) I'll rename it. for now. But I have on my todo to make that page so that it doesn't rely on the Pool form so that we can delete the controller and use the template directly from routing

**Pull request merged**